spec/rubocop/cop/sorbet/signatures/allow_incompatible_override_spec.rb

```diff
@@ -3,24 +3,39 @@
 require "spec_helper"
 
 RSpec.describe(RuboCop::Cop::Sorbet::AllowIncompatibleOverride, :config) do
-  def message
-    "Usage of `allow_incompatible` suggests a violation of the Liskov Substitution Principle. "\
-    "Instead, strive to write interfaces which respect subtyping principles and remove `allow_incompatible`"
-  end
+  let(:message) { RuboCop::Cop::Sorbet::AllowIncompatibleOverride::MSG }
 
   it("disallows using override(allow_incompatible: true)") do
     expect_offense(<<~RUBY)
       class Foo
-        sig(a: Integer).override(allow_incompatible: true).void
+        sig(a: Integer).override(allow_incompatible: true, something: :unrelated).void
```

Comment:
This test shouldn't have extra keywords:

```suggestion
        sig(a: Integer).override(allow_incompatible: true).void
```

I added the test below to test that separately